bitcoin/src/lib.rs

```diff
@@ -121,9 +121,11 @@ use core2::io;
 pub use crate::address::{Address, AddressType};
 pub use crate::amount::{Amount, Denomination, SignedAmount};
 pub use crate::blockdata::block::{self, Block};
+pub use crate::blockdata::fee_rate::FeeRate;
 pub use crate::blockdata::locktime::{self, absolute, relative};
 pub use crate::blockdata::script::{self, Script, ScriptBuf};
 pub use crate::blockdata::transaction::{self, OutPoint, Sequence, Transaction, TxIn, TxOut};
+pub use crate::blockdata::weight::Weight;
```

Comment:
Maybe ` as BlockWeight` to improve clarity? `bitcoin::Weight` looks a bit strange.

Reply:
FWIW, I would be confused if it's `TxWeight` or `BlockWeight`.

Reply:
I think the docstring does a great job of this clarification already

> /// Represents block weight - the weight of a transaction or block.
> ///
> /// This is an integer newtype representing weigth in `wu`. It provides protection against mixing
> /// up the types as well as basic formatting features.

Reply:
I don't think "blockweight" makes sense. Weight can apply to transactions just as well (and this is the more common case).

I think `bitcoin::Weight` looks fine. "weight" is a standard unit in Bitcoin.

Reply:
In my mind calling transaction weight "block weight" was OK because it refers to how much weight does it take in block. But OK, looks like simple `Weight` is fine.